.github/workflows/CI.yml

```
@@ -14,5 +14,13 @@ jobs:
     - uses: actions/checkout@v4
     - name: Build
       run: cargo build --verbose
+    - name: Add fmt
+      run: rustup component add rustfmt
+    - name: Add Clippy
+      run: rustup component add clippy
+    - name: Format
+      run: cargo fmt --all -- --check
+    - name: Clippy
+      run: cargo clippy
```

чтобы линты показывались как ошибки:
```suggestion
      run: cargo clippy -- -D warnings
```